extensions/wikia/AdEngine/js/video/videoSettings.js

```diff
@@ -30,6 +31,10 @@ define('ext.wikia.adEngine.video.videoSettings', [
 				return params.moatTracking;
 			}
 
+			if (typeof params.moatTracking === 'string') {
+				sampling = instantGlobals[params.moatTracking];
```

Comment:
should we take InstantGlobal name from params? Or should we simply use value from: `wgAdDriverPorvataMoatTrackingSampling` InstantGtlobal?